changelog/unreleased/add-drives-endpoint-and-userfilter.md

```diff
@@ -4,4 +4,5 @@ We have added `$expand=drives` to the `/users/{id}/` endpoint
 using the user filter implemented in reva.
 
 https://github.com/owncloud/ocis/pull/4072
-https://github.com/cs3org/reva/pull/3046
\ No newline at end of file
+https://github.com/cs3org/reva/pull/3046
+https://github.com/cs3org/reva/pull/4323
```

Comment:
inexistent link

Reply:
fixed, thanks

Reply:
probably you deleted the correct link (3046). Wrong link(4323) left

Reply:
fixed again 😄